### PR TITLE
clarify unpair copy

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

**Comment by pd0wm** [comment]
Don't check in your .swp file. Maybe add to gitignore?

File: src/components/Dashboard/DeviceSettingsModal.js [hunk 1]
```diff
@@ -354,7 +354,7 @@ class DeviceSettingsModal extends Component {
           { this.props.device.prime &&
             <div className={ classes.unpairWarning }>
               <WarningIcon />
-              <Typography>unpairing will also cancel your prime subscription</Typography>
+              <Typography>Unpairing will also cancel the device's comma prime subscription.</Typography>
```
Comment (gregjhogan) [inline src/components/Dashboard/DeviceSettingsModal.js]:
```suggestion
              <Typography>Unpairing will also cancel the comma prime subscription for this device.</Typography>
```

Comment (adeebshihadeh) [reply src/components/Dashboard/DeviceSettingsModal.js]: I think the current wording is better, but don't prefer it strongly. @pd0wm want to break the tie?

Comment (pd0wm) [reply src/components/Dashboard/DeviceSettingsModal.js]: Think @gregjhogan wording makes more sense. The device doesn't own the prime subscription, the user does.

**Pull request merged**